.changeset/light-eagles-promise.md

```diff
@@ -0,0 +1,13 @@
+---
+'@envelop/core': minor
+---
+
+allow masking validation and parse errors with `useMaskedErrors`.
+
+```ts
+useMaskedErrors({ onValidate: true, onParse: true });
+```
+
+This option is disabled by default as validation and parse errors help the API consumer instead of leaking secret information.
```

Comment:
```suggestion
This option is disabled by default as validation and parse errors are expected errors that help the API consumer instead of leaking secret information.
```